demisto_sdk/commands/common/tools.py

```diff
@@ -1826,6 +1828,36 @@ def get_content_id_set() -> dict:
     return requests.get(OFFICIAL_CONTENT_ID_SET_PATH).json()
 
 
+def download_content_graph(
+    output_path: Path, marketplace: MarketplaceVersions = MarketplaceVersions.XSOAR
+) -> Path:
+    """Getting the Content Graph from official content's bucket"""
+    if output_path.is_dir():
+        output_path = output_path / f"{marketplace.value}.zip"
+    with open(output_path, "wb") as f:
+        f.write(
+            requests.get(
+                f"{OFFICIAL_CONTENT_GRAPH_PATH}/{marketplace.value}.zip"
+            ).content
+        )
+    return output_path
+
+
+def get_latest_upload_flow_commit_hash() -> str:
+    """Getting the latest commit hash of the upload flow from official content's bucket
+
+    Returns:
+        str: the last commit hash of the upload flow
+    """
+    response_json = requests.get(OFFICIAL_INDEX_JSON_PATH).json()
+    if not isinstance(response_json, dict):
+        raise ValueError("The index.json file is not in the expected format")
```

Comment:
```suggestion
        raise ValueError(f"The index.json file is not in the expected format: {response_json}")
```

So people know what's going on without having to re-run in debug mode